docs/exceptions/exceptions-logs.md

```diff
@@ -52,4 +56,13 @@ The table below indicates which attributes should be added to the
 Same as [Trace Semantic Conventions for Exceptions - Stacktrace
 Representation](exceptions-spans.md#stacktrace-representation).
 
+## Guidelines for Recording Exceptions in Spans, Logs, and Events
+
+When it comes to recording exceptions, it's crucial to distinguish between Spans, Logs, and Events:
+
+* Spans and Events: Exception recording in Spans and Events follows a similar pattern. Both utilize the event name `exception`. Exceptions MAY be recorded within Span Events if they occur during the span's lifecycle.
+* Logs: Exceptions SHOULD be recorded in logs when utilizing the log bridge API to map application logs to OpenTelemetry logs.
+
+Note that in all cases the exception data is stored in attributes.
```

Comment:
Can we entertain the idea of recording stack trace in the payload and not in the attribute to avoid all the common problems (parsing, truncation, etc) ?

Reply:
@lmolkova  Can you elaborate a bit on how recording the stacktrace in the Event Payload (LogRecord Body) address those issues? Even within the `any` type field, the data would still have to be put into the standard primitive types.

Reply:
Check out https://github.com/open-telemetry/opentelemetry-specification/issues/2839

Reply:
@trask thanks for the pointer. I guess we were taking the simplistic approach of keeping the `exception` event consistent with `exception` spans and `exception` logs. However, I can see that moving the exception data to Event body/payload would allow us to have richer data in the Event. For eg., in addition to the structured vs plaintext stacktrace representation, we could add `context` or `additonal_data` to the Event to capture arbitrary objects which is not possible with attributes which only allow primitive types currently.